client/src/pages/Login/index.js

```diff
@@ -1,6 +1,90 @@
+import { Button, Form, Input } from 'antd';
+import PropTypes from 'prop-types';
 import React from 'react';
+import { Link } from 'react-router-dom';
+import loginImg from '../../assets/loginImg.png';
 import './style.css';
 
-const Login = () => <div className="Login" />;
+class Login extends React.Component {
+  state = {
+    email: '',
+    password: '',
+    error: false,
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { email, password } = this.state;
+    fetch('/api/v1/login', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        email,
+        password,
+      }),
+    })
+      .then((result) => {
+        const {
+          history: { push },
+        } = this.props;
+        push(`/profile${result.data.message}`);
```

Comment:
Why data.message? Just to his profile,
This function need to do one more thing
make the context do another request to check the user auth state and set his state